premake5.lua

```diff
@@ -659,3 +659,57 @@ if (os.istarget("macosx")) then
     end
 
 end
+
+-- VST3 PLUGIN --
```

Comment:
Can you adjust this comment? Thank you!

Reply:
You are right, I've missed it totally. Fixed